torch/_inductor/codegen/triton.py

```diff
@@ -2409,7 +2410,7 @@ def candidate_tilings(node):
         deps = [
             dep
             for dep in itertools.chain(rw.reads, rw.writes)
-            if dep.name not in V.graph.removed_buffers
+            if dep.name not in V.graph.removed_buffers and isinstance(dep, MemoryDep)
```

Comment:
Curious, why do we want to ignore other `dep` types than `MemoryDep` here?

Reply:
Practically, because later we use `dep.index` which doesn't exist on StarDeps. Conceptually my understanding is that StarDep accesses also won't contribute suggestions for tiling since they don't have indexing information.

Added a comment